examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST (works only if running on localhost)</h4>
+<form method="post"
+      action-xhr="http://iframe.localhost:8000/form/redirect-to/post"
+      target="_blank">
```

Comment:
Ok, I am moving the redirection handling out of XHR and into `amp-form` since it is currently its only client, if in future others wanted to use redirection we can rethink this.